src/Infrastructure/LeanCode.Serialization/JsonConverters/JsonLaxDateOnlyConverter.cs

```diff
@@ -1,16 +1,26 @@
+using System.Buffers;
 using System.Globalization;
+using System.Text;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace LeanCode.Serialization;
 
 public class JsonLaxDateOnlyConverter : JsonConverter<DateOnly>
 {
+    private static readonly UTF8Encoding encoder = new(false);
+
     public override DateOnly Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {
-        return reader.GetString() is string s
-            ? DateOnly.FromDateTime(DateTime.Parse(s, CultureInfo.InvariantCulture))
-            : default;
+        var value = reader.ValueSpan;
```

Comment:
What if the reader was constructed from `ReadOnlySequence<byte>` instead? You must check if `HasValueSequence` is false before accessing `ValueSpan` property (and if it's not, use `ValueSequence` property instead).

But actually, you probably want to use `CopyString` instead, see the comment about (un)escaping below.